code/game/gamemodes/miniantags/slaughter/slaughter.dm

```diff
@@ -42,8 +42,15 @@
 							Pulling a dead or unconscious mob while you enter a pool will pull them in with you, allowing you to feast and regain your health. \
 							You move quickly upon leaving a pool of blood, but the material world will soon sap your strength and leave you sluggish. </B>"
 
+	loot = list(/obj/effect/decal/cleanable/blood, \
+				/obj/effect/decal/cleanable/blood/innards, \
+				/obj/item/organ/internal/heart/demon)
+	del_on_death = 1
+	var/death_message = "<span class='danger'>The generic slaughter demon screams in anger because this is the default death message!</span>"
```

Comment:
Not var/death_message, deathmessage
It's an already-defined var(and includes the danger span class)